# play_with_input

作成過程のメモ

Traceback (most recent call last):
  File "/Users/kashimataichi/play_python/play_with_input/addition.py", line 5, in <module>
    answer = input(num_first, '+', num_second, '=')
TypeError: input expected at most 1 arguments, got 4
DarkDream:play_python kashimataichi$
→input関数の引数は1つだけ指定する
